### PR TITLE
Allow the oauth landing page template to be customizable

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Allow the oauth landing page to be customizable
## Impacts

OAuth in the portal and the admin
## NEWS file entries

Enhancements
++++++++++++
- Allow the oauth landing page template to be customizable
## Delete branch after merge

NO

### CODE REVIEW

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/Authentication/OAuth.pm [hunk 1]
```diff
@@ -12,16 +12,31 @@ Form definition to create or update an authentication portal module.
...
+has_field 'landing_template' =>
+  (
+   type => 'Text',
+   label => 'Landing template',
+   required => 1,
+   tags => { after_element => \&help,
+             help => 'The template to use for the signup' },
+  );
+
+sub BUILD {
```
Comment (julsemaan) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/Authentication/OAuth.pm:37]: doc header

**Comment by julsemaan** [comment]
Just one small touchup to do and its good to merge

**Comment by jrouzierinverse** [comment]
done

**Comment by fdurand** [comment]
Touchup has been done, let's merge

**Pull request merged**